### DISCUSSION TITLE
how to draw a branching arrow

### DISCUSSION THREAD

**Discussion by daiskaneko** [jgraph/drawio#3065] [Q&A]
Hello,

Could you tell me how to draw a branching arrow as following image?
![image-260](https://user-images.githubusercontent.com/105411147/191700779-2693df52-b34e-4d96-8264-b42b763daf65.png)

Thanks,

**Comment by xfudox** [comment]
I think there's no way to create a _single_ branching arrow, but you can either
- create a first arrow from the top box to one of the bottom ones and a second one from the branching point to the other
- create two arrows both starting from the top box and overlapping for the first part of the path and than pointing each one to its bottom box

**Reply by daiskaneko** [reply]
Thank you for your comment.
